src/System.Security.Cryptography.Cng/src/Microsoft/Win32/SafeHandles/NCryptSafeHandles.cs

```diff
@@ -319,6 +361,15 @@ internal bool ReleaseNativeWithNCryptFreeObject()
     /// </summary>
     public sealed class SafeNCryptKeyHandle : SafeNCryptHandle
     {
+        public SafeNCryptKeyHandle()
+        {
+        }
+
+        public SafeNCryptKeyHandle(IntPtr handle, SafeHandle parentHandle)
```

Comment:
I don't see this ctor documented on msdn or in desktop.  Is it new public surface area?  Should it be internal?